### ISSUE TITLE
Remove `timetzdata` tag from Windows builds when `golang/go#38453` merges

### ISSUE THREAD

**Issue by danxmoran** [influxdata/influxdb#22453]
golang/go#38453 tracks updating `time.LoadLocation` to use native Windows APIs. If it is implemented, we can remove the `timetzdata` tag from our Windows builds to stop bundling a copy of Go's tz database in our binaries.

**Comment by timhallinflux** [comment]
https://github.com/influxdata/influxdb/issues/21756 is this related?

**Comment by danxmoran** [comment]
@timhallinflux yes. #21756 was fixed by adding a work-around setting to our Windows build process. This ticket tracks removing the work-around once upstream Go merges & releases native timezone support for Windows.
